core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConfig.java

```diff
@@ -134,6 +134,17 @@
     @ConfigItem(defaultValue = "10m")
     Duration hangDetectionTimeout;
 
+    /**
+     * The type of test to run, this can be either:
+     *
+     * quarkus-test: Only runs {@code @QuarkusTest} annotated test classes
+     * unit: Only runs classes that are not annotated with {@code @QuarkusTest}
+     * all: Runs both, running the unit tests first
+     *
+     */
+    @ConfigItem(defaultValue = "all")
+    TestType testType;
```

Comment:
Can be just `type`? To avoid the redudant `test` in `quarkus.test.test-type`, since the config root is already `test`?

Reply:
done